.github/workflows/trivy.yml

```diff
@@ -0,0 +1,24 @@
+name: trivy check
+on:
+  push:
+    branches:
+    - main
```

Comment:
We don't have a main branch, but a master branch.

Should this run on push or pull_request? The CI error at https://github.com/BitBoxSwiss/bitbox-wallet-app/pull/2768#issuecomment-2178124434 seems to say pull_request. And we'd want to see problems on pull requests right?

Reply:
Fixed to master, now also scans PR. Had to add `permissions:security-events: write`